src/psyclone/psyir/nodes/dynamic_omp_task_directive.py

```diff
@@ -268,12 +253,24 @@ def _handle_proxy_loop_index(self, index_list, dim, index,
                         ArrayReference
         :param index: The index that is a proxy to a parent loop variable.
         :type index: :py:class:`psyclone.psyir.nodes.Reference`
-        :type private_list: List[:py:class:`psyclone.psyir.nodes.Reference`]
-        :param firstprivate_list: The firstprivate References for this
-                                  task.
-        :type firstprivate_list: List[
-                                 :py:class:`psyclone.psyir.nodes.Reference`]
-        :param in_list: The input References for this task.
+        :param clause_lists: The namedtuple containing the lists storing the
+                             clauses.
+        :type clause_lists: namedtuple(
```

Comment:
So much for my plan to shorten docstrings although I do think this is nice so thank you.